custom_model_runner/datarobot_drum/runtime_parameters/runtime_parameters_schema.py

```diff
@@ -0,0 +1,35 @@
+"""
+Copyright 2023 DataRobot, Inc. and its affiliates.
+All rights reserved.
+This is proprietary source code of DataRobot, Inc. and its affiliates.
+Released under the terms of DataRobot Tool and Utility Agreement.
+"""
+from enum import Enum
+
+import trafaret as t
+
+
+class RuntimeParameterTypes(Enum):
+    STRING = "string"
+    CREDENTIAL = "credential"
+
+    @classmethod
+    def all(cls):
+        return [v.value for k, v in cls.__dict__.items() if isinstance(v, cls)]
```

Comment:
I thought the built in Enum class supported iteration so this wasn't really necessary because you can have code like "if given_type in EnumClass". Or if you really want a list of all the types you can do "list(EnumClass)"